README.md

```diff
@@ -73,28 +54,55 @@ builder.Services.AddSingleton<ISinch>(x => new SinchClient(
     builder.Configuration["Sinch:ProjectId"]));
 ```
 
+To configure Conversation or Sms hosting regions, and any other additional paramters, use [`SinchOptions`](https://github.com/sinch/sinch-sdk-dotnet/blob/main/src/Sinch/SinchOptions.cs):
+
+```csharp
+var sinch = new SinchClient(
+    configuration["Sinch:KeyId"],
+    configuration["Sinch:KeySecret"],
+    configuration["Sinch:ProjectId"],
```

Comment:
I've always wondered why the `ProjectId` comes third: in the dashboard, the topmost object is the Project Id. Inside a project you then create an access key, so I would have thought the same order would be reflected in the first constructor a user will see